server/src/main/java/org/elasticsearch/action/admin/cluster/snapshots/create/CreateSnapshotResponse.java

```diff
@@ -45,6 +48,10 @@
     CreateSnapshotResponse() {
     }
 
+    void setSnapshotInfo(SnapshotInfo snapshotInfo) {
```

Comment:
Can we use the existing constructor in the only place where this is called (test) and remove this new method?

Reply:
Done.